pages/index.js

```diff
@@ -98,19 +77,12 @@ export default function HomePage() {
         <StyledSpan>100</StyledSpan>
       </StyledWrapper>
       <StyledButton type="submit">Save</StyledButton>
-      <StyledList>
-        {tensionEntry.map((entry) => {
-          const { id, date, tensionLevel } = entry;
-          return (
-            <StyledListItem key={id}>
-              {date} - Tension Level: {tensionLevel}%
-            </StyledListItem>
-          );
-        })}
-      </StyledList>
-      <StyledLink href="/emotions">The 7 basic emotions</StyledLink>
+      <StyledNav>
+        <StyledLink href="/emotions">The 7 basic emotions</StyledLink>
+        <StyledLink href="/emotion-records">Emotion records</StyledLink>
+      </StyledNav>
```

Comment:
I am not a fan to place navigation link / elemenst  inside a `<form>` element - even if it's perhaps easier to style. 

Navigation links are intended to direct users to different pages of a website. They are not part of the data submission process and thus don't semantically belong inside a form.  🫠

Reply:
Thank you very much for your detailed and helpful feedback on our code! We implemented most of your suggestions happily. We also agree with your suggestions about semantic StyledComponents-Names and &:hover effects. We decided to change these in a future update together with a well thought concept for our own StyledComponents "library".